data/human/culture conversations.txt

```diff
@@ -344,11 +344,22 @@ mission "Forest Fire"
 mission "Local Politics"
 	minor
 	source
-		government "Free Worlds"
 		attributes "frontier"
```

Comment:
Any planet with "south pirate" also has the "south" attribute. Therefore, we can simplify the offer conditions a bit.
```suggestion
		attributes "frontier"
		attributes "south"
```